docs/transpile/index.mdx

```diff
@@ -0,0 +1,42 @@
+1. `layout` - This pass applies a *layout*, mapping/assigning the virtual qubits in your circuit to the physical qubits of a backend.
+1. `routing` - This pass runs after a layout has been applied and will inject gates (i.e., SWAPs) in the original circuit in order to make it compatible with the backend's connectivity/coupling map.
+1. `translation` - This pass translates the gates in the circuit to the backend's basis set of instructions.
+1. `optimization` - This pass runs an optimization loop to find more efficient decompositions of your quantum circuit until a condition is met (such as a fixed depth).
+1. `scheduling` - This pass is for any hardware-aware optimizations that improves the efficiency of the executed microwave pulse instructions.
+
+ If you decide to customize your own transpilation workflow, we suggest using these stages as a guideline during development.
+
+
+
+## Default Transpilation
+The basic usage of the transpiler is to call the [`qiskit.compiler.transpile()`](../api/qiskit/compiler#compilation-routines) function.  This will generate and run one of the preset `StagedPassManager`s based on, among other options, an `optimization_level` flag which can be set to either 0, 1, 2, or 3.  Higher levels generate more optimized circuits, at the expense of longer transpilation times.
+
+- `optimization_level=0`: Just maps the circuit to the backend, with no explicit optimization.
+- `optimization_level=1`: Maps the circuit, but also does some lightweight optimizations by collapsing adjacent gates.
+- `optimization_level=2`: Medium-weight optimization, including a noise-adaptive layout and a gate-cancellation procedure based on gate commutation relationships.
+- `optimization_level=3`: Heavy-weight optimization, which in addition to previous steps, resynthesizes two-qubit blocks of gates in the circuit.
+
+If you'd like to read more about the preset pass managers or the transpiler stages, look to the section on [Transpiler Stages & Preset PassManagers](passmanagers.ipynb).
+
+
+## Customizing Passes
+For more advanced users, you can customize your own set of `PassManager` and `StagedPassManager` objects and determine the ordering in which each stage is run.  This can dramatically change the final circuit which is output. If fact, oftentimes a bespoke approach to transpiling a quantum algorithm produces more efficient error suppression than the default approach.  This involves rewriting quantum circuits to match hardware constraints and suppressing the effects of noise.  The flow of logic for this tool chain is quite customizable and need not be linear.  The transpilation process can even prepare iterative loops, conditional branches, and other complex behaviors.  A good starting place when developing a set of custom passes is by looking to the default sequence of transformations:
```

Comment:
```suggestion
For more advanced users, you can customize your own set of `PassManager` and `StagedPassManager` objects and determine the ordering in which each stage is run.  This can dramatically change the final output circuit. In fact, oftentimes a bespoke approach to transpiling a quantum algorithm produces more efficient error suppression than the default approach.  This involves rewriting quantum circuits to match hardware constraints and suppressing the effects of noise.  The flow of logic for this tool chain is quite customizable and need not be linear.  The transpilation process can even prepare iterative loops, conditional branches, and other complex behaviors.  A good starting place when developing a set of custom passes is to look at the default sequence of transformations:
```

Reply:
What about "to examine" rather than "to look at"?

Reply:
Yep! That's better :)

Reply:
Done!  And addressed the rest of your suggestions.